src/worker.cc

```diff
@@ -517,6 +517,12 @@ void parse_contours(const rapidjson::Document& doc,
  */
 void from_json(rapidjson::Document& doc, Options::Action action, Api& api) {
   // if its a pbf request we want to keep the options and clear the rest
+
+  // lambda fuction to add warnings to api object
+  auto add_warning = [](Api& api, const std::string& warning_text) {
+    api.mutable_info()->mutable_warnings()->Add()->set_description(warning_text);
+  };
```

Comment:
seems like we can use this anywhere we want to add  a warning potentially? maybe we should just add this as a function in worker.h so anyone can call it from anywhere

Reply:
Okay. I've pushed this update in the latest commits.